docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -1,10 +1,450 @@
 [[fleet-api-docs]]
-= {fleet} APIs
+= {kib} {fleet} APIs
 
 experimental[]
```

Comment:
Should this be `beta`? Or should references to the Fleet API in [`agent-policies.asciidoc`](https://github.com/elastic/observability-docs/pull/2269/files#diff-977a9a6da7dcf823ccc6e8295230c9bd5a69b49c9cde8ba62059561cd3ba11a2R118) be `experimental`? Or are these not the same thing?

Reply:
The API preview feature described in agent-policies.asciidoc is beta, not the API. I guess this is a bit ambiguous, but I'm not sure how to make it more clear.

Reply:
I'm not sure if it really matters. The "consistency alarm" just went off in my brain when I saw them.

Reply:
This is a _good thing_! 🥇